So
I
granted
all
privileges
to
root
in
MySQL
So
after
running
this
i
get
:
But
after
running
my
code
on
Pycharm
which
accesses
a
local
DB
","
I'm
getting
this
:
I'm
pretty
new
to
Python
and
MySQL
","
so
I'm
gonna
need
pretty
simple
yet
detailed
advice
if
anyone
is
willing
to
help
.
Thanks
!
